### PR TITLE
feat: Edit webhook URL

### LINKED ISSUE #1878: feat: Ability to edit out going address of API Channel
At the moment on chatwoot, After creating an API channel, there isn't the ability to edit the webhook URL

### Acceptance: 

- [ ] As an admin, I should be able to come back and edit the webhook URL for an existing API channel
- [ ] Verify the event payloads are correctly delivered to the new webhook URL post update

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes #1878

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/integrations/EditWebHook.vue [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    endPoint: {
+      required,
+      minLength: minLength(7),
+      url,
+    },
+  },
+  methods: {
+    showAlert() {
+      bus.$emit('newToastMessage', this.addWebHook.message);
+    },
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/settings/integrations/EditWebHook.vue:94]: Please use the mixin.

File: app/javascript/dashboard/routes/dashboard/settings/integrations/EditWebHook.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      this.$v.endPoint.$reset();
+    },
+    async addWebhook() {
+      this.addWebHook.showLoading = true;
+      try {
+        await this.$store.dispatch('webhooks/update', {
+          webhook: { url: this.endPoint },
+          id: this.webhookId,
+        });
+        this.addWebHook.showLoading = false;
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/settings/integrations/EditWebHook.vue:106]: this can be moved to `finally` block

**Comment by muhsin-k** [comment]
@pranavrajs Requested changes are done

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by pranavrajs — DISMISSED** [review]
Please add specs for mutations/actions. Otherwise looks good.

**Review by pranavrajs — DISMISSED** [review]

**Comment by pranavrajs** [comment]
@muhsin-k Seems like there is a conflict, can you please fix it?

**Pull request merged**